tutorials/QML_with_Classiq/QML_with_Classiq.ipynb

```diff
@@ -0,0 +1,1203 @@
+{
```

Comment:
<p>Seems like ReviewNB issue as here I also didn't call it "Details"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Same</p>